.github/workflows/pr-ebpf-selfhosted.yaml

```diff
@@ -0,0 +1,30 @@
+name: pr-ebpf
+on: 
+  workflow_dispatch:
```

Comment:
I assume this is what makes it trigger manually right? I was reading the docs but didn't see the prompt like shown https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Will I see it once we merge it?

Reply:
Also who can trigger this? The maintainers of this repo or anyone in Aqua Github org?

Reply:
I merged this in my personal fork to confirm, yes it will show once merged:

![Screen Shot 2021-04-21 at 4 06 42 PM](https://user-images.githubusercontent.com/10857349/115614378-a70a8280-a2bb-11eb-9617-dd1a85771652.png)

And I believe maintainers can trigger it.

Reply:
Also I renamed the self hosted workflow files.